README.md

```diff
@@ -36,6 +36,8 @@ $ stack --version # we'll want to know this version if you run into trouble
 $ stack build && stack exec tests && stack exec unison
 ```
 
+To run a local codebase-ui while building from source, you can use the `/dev-ui-install.sh` script. It will download the latest release of the codebase-ui and put it in the expected location for the unison executable created by `stack build`. When you start unison, you'll see a url where the codebase-ui is running. 
+
 See [`development.markdown`](development.markdown) for a list of build commands you'll likely use during development.
```

Comment:
This use of `dev-ui-install.sh` works well when starting unison with `stack exec unison` from the repo root on disk.

```
$ tree
.
├── prettyprintdemo
├── runtests
├── tests
├── transcripts
├── ui
│   ├── index.html
│   └── static
│       ├── 145e4ac7976fbd199e74.woff2
│       ├── 192496840904816ec56e.svg
│       ├── 73d88d2ca065037a4d55.woff2
│       ├── e2e12d2e9288439ecb06.woff2
│       ├── favicon.ico
│       └── main.7aa2e19635a11485bd22.js
└── unison

$ pwd
/Users/pdejoux/dev/src/unison/unison/.stack-work/install/x86_64-osx/d91a3831cdbede648511633d77fcf1ba6816be4a7a56348baac333e06c424a8c/8.10.4/bin
```

If I've installed unison by doing `stack install unison-parser-typechecker:unison` then I can put the `codebase-ui` in the same relative position and unison will pick it up.

```
$ tree
.
├── ui
│   ├── index.html
│   └── static
│       ├── 145e4ac7976fbd199e74.woff2
│       ├── 192496840904816ec56e.svg
│       ├── 73d88d2ca065037a4d55.woff2
│       ├── e2e12d2e9288439ecb06.woff2
│       ├── favicon.ico
│       └── main.7aa2e19635a11485bd22.js
└── unison

$ pwd
/Users/pdejoux/.local/bin
```